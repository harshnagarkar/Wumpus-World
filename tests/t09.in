cabfdeg-h
-abcdefgh
id
5
