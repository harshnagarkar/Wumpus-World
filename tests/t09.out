 c a b
 f d e
 g - h

 c a b
 f d e
 - g h

 c a b
 - d e
 f g h

 - a b
 c d e
 f g h

Total nodes: 27, max depth is: 3
